presto-docs/src/main/sphinx/release/release-0.255.rst

```diff
@@ -0,0 +1,39 @@
+=============
+Release 0.255
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix an issue where regular expression functions were not interruptible and could keep running for a long time after the query they were a part of failed.
+* Add support to interrupt runaway splits blocked in known situations. The interrupt timeout can be configured by the configuration property ``task.interrupt-runaway-splits-timeout``. The default value is ``600s``.
+* Added :func:`array_normalize` function to normalize an array by dividing each element by the p-norm of it.
+
+JDBC Connector Changes
+______________________
+* Added connection param ``protocols`` that allows a user to specify which HTTP protocols the Presto client is allowed to use.
+
+Cassandra Connector Changes
+___________________________
+* Add support for Cassandra ``SMALLINT``, ``TINYINT`` and ``DATE`` types.
+
+Druid Connector Changes
+_______________________
+* Add support for querying non-lowercase table names in Druid connector (:pr:`15920`).
+
+Elasticserarch Changes
+______________________
+* Fix to avoid ``NullPointerException`` when there is an unsupported data type column in the Object field.
+
+Hive Changes
+____________
+* Fix the import of ``SMALLINT`` in ParquetReader.
+* Revert Support Dwrf Sequence Ids in Writer.
```

Comment:
This does not read as a sentence. Lets just use the language in 0.254.1 release note - Revert non-backward-compatible DWRF writer updates (#15028)

Reply:
Do you mean this PR? https://github.com/prestodb/presto/pull/16037

Reply:
@mayankgarg1990 #15028 is about reverting zstd version.

Reply:
They might have quoted the wrong PR - so yes - what you have is the right PR

Reply:
Shall we fix the 0.254.1 release notes as well?

Reply:
I will fix that as well.